crates/db_views/src/private_message_view.rs

```diff
@@ -66,6 +66,10 @@ fn queries<'a>() -> Queries<
       )
     }
 
+    if let Some(i) = options.creator_id {
+      query = query.filter(private_message::creator_id.eq(i))
```

Comment:
Whoops, just read @Nutomic 's comment. Look at the filters above, this conflicts with some of them.

If its unread only, then this simple filter is fine (IE it should not show messages created by you.) So move this one into the unread block.

But if its `All`, then it should check if the creator is this creator OR you.

Reply:
Ok yeah, should work the intended way now! Thanks for the feedback!